bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusPushButtonHandler.java

```diff
@@ -42,13 +46,16 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * The {@link NikobusPushButtonHandler} is responsible for handling Nikobus push buttons.
+ * The {@link NikobusPushButtonHandler} is responsible for handling Nikobus push
+ * buttons.
  *
  * @author Boris Krivonog - Initial contribution
- * @author Wouter Denayer - support for module addresses as seen in the Niko PC tool
+ * @author Wouter Denayer - support for module addresses as seen in Niko PC app
+ * 
  */
 @NonNullByDefault
 public class NikobusPushButtonHandler extends NikobusBaseThingHandler {
+    @NonNullByDefault
```

Comment:
AFAIK this is redundand since `NikobusPushButtonHandler` is already marked with  `@NonNullByDefault`